rskj-core/src/main/java/co/rsk/peg/feeperkb/FeePerKbStorageProviderImpl.java

```diff
@@ -32,13 +34,13 @@ private void saveFeePerKb() {
     }
 
     @Override
-    public Coin getFeePerKb() {
+    public Optional<Coin> getFeePerKb() {
         if (feePerKb != null) {
-            return feePerKb;
+            return Optional.of(feePerKb);
         }
 
         feePerKb = bridgeStorageAccessor.safeGetFromRepository(FEE_PER_KB.getKey(), BridgeSerializationUtils::deserializeCoin);
-        return feePerKb;
+        return Optional.of(feePerKb);
```

Comment:
If the response if the storage accessor is null this will probably fail. Check if using Optional.ofNullable helps